sigopt_example.m

```diff
@@ -1,40 +1,51 @@
-% 
-%
-% define dimensions
-%
-% make connection object
-conn = py.sigopt_matlab.get_connection()
+import sigopt.Connection
 
-% dimension 1
-x_bounds = py.dict(pyargs('min', 0, 'max', 1))
-x_parameter = py.dict(pyargs('name', 'x', ...
-                             'type', 'double', ...
-                             'bounds', x_bounds))
-                        
-% dimension 2                
-y_bounds = py.dict(pyargs('min', 0, 'max', 1))
-y_parameter = py.dict(pyargs('name', 'y', ...
-                             'type', 'double', ...
-                             'bounds', y_bounds))
+conn = Connection('<YOUR API TOKEN HERE>')
 
-                    
-% define parameters and name variables
-parameters = py.list({x_parameter, y_parameter})
-name = 'Franke Optimization (MATLAB)'
+experiment = conn.experiments().create(struct( ...
```

Comment:
what is the typical matlab indentation style? i can update

yeah - that would be nice but ideally we want the API client to be as thin as possible. all the validation should be done serverside. that's what we're doing in the python client now too (and that's why this just delegates to the python client). not sure if there's any way to add in the construct you are describing if we want all the keys to be dynamically defined (but happy to update if that is possible)